File: solutions/013/013.go
package main

import (
	"fmt"
	"io/ioutil"
	"strconv"
	"strings"
)

func main() {
	bs, _ := ioutil.ReadFile("./resources/numbers.txt")
	lines := strings.Split(string(bs), "\n")
	sum := 0
	for _, line := range lines {
		n, _ := strconv.Atoi(line[:11])
		sum += n
	}
	fmt.Println(strconv.Itoa(sum)[:10])
}

File: solutions/013/013.c
#include <stdio.h>
#include <string.h>

int main() {
    FILE *fp = fopen("./resources/numbers.txt", "r");
    char line[60] = {0};
    unsigned long sum = 0;
    unsigned long n = 0;

    while (fgets(line, 80, fp)) {
        *(line + 11) = 0;
        sscanf(line, "%lu", &n);
        sum += n;
    }

    while (sum >= 1e10)
        sum /= 10;

    printf("%lu\n", sum);
}
